management/server/grpcserver.go

```diff
@@ -16,7 +16,7 @@ import (
 
 // Server an instance of a Management server
 type Server struct {
-	accountManager *AccountManager
+	accountManager *DefaultAccountManager
```

Comment:
I suggest keeping the AccountManager interface here, then we can easily replace it with some other implementation if needed. 
It will also be easier to mock in tests
That is the whole purpose of interfaces :)